Basic_viewer/doc/Basic_viewer/PackageDescription.txt

```diff
@@ -0,0 +1,71 @@
+/// \defgroup PkgBasicViewerRef Basic Viewer Reference
+
+/// \defgroup PkgBasicViewerClasses Classes
+/// \ingroup PkgBasicViewerRef
+
+/*!
+\addtogroup PkgBasicViewerRef
+\cgalPkgDescriptionBegin{Basic Viewer,PkgBasicViewer}
+\cgalPkgPicture{TODO.png}
+\cgalPkgSummaryBegin
+\cgalPkgAuthor{Guillaume Damiand, Mostafa Ashraf}
+\cgalPkgDesc{This package implements basic viewer class.}
+\cgalPkgManuals{Chapter_Basic_viewer,PkgBasicViewerRef}
+\cgalPkgSummaryEnd
+\cgalPkgShortInfoBegin
+\cgalPkgSince{6.0}
+\cgalPkgBib{cgal:XXX}
+\cgalPkgLicense{\ref licensesLGPL "LGPL"}
+\cgalPkgShortInfoEnd
+\cgalPkgDescriptionEnd
+
+\cgalClassifedRefPages
+
+\cgalCRPSection{Classes}
+- `CGAL::Drawing_functor<DS, Vertex_descriptor, Edge_descriptor, Face_descriptor>`
+- `CGAL::Drawing_functor_with_volume<DS, Vertex_descriptor, Edge_descriptor, Face_descriptor, Volume_descriptor>`
+- `CGAL::Graphic_storage<BufferType>`
+- `CGAL::Basic_viewer_qt<BufferType>`
+- `CGAL::QApplication_and_basic_viewer<BufferType>`
+
+\cgalCRPSubsection{Draw a Graphic Storage}
+
+- `CGAL::draw_graphic_storage()`
```

Comment:
I prefer draw_graphic_storage. This function draw the graphic storage in parameter.

Reply:
and simply `draw()` is not possible?

Reply:
You suggest  "from" because what is in the storage is drawn, but this argument would be similar for any function that takes a container of things as argument.   For example we document "vertices	the vertices of the patches to be faired"  and we do not write "the range of vertices".

Reply:
Conclusion: we all agree to keep the current name?

Reply:
agreed by @afabri and @sloriot